packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantSkeleton/RestaurantContentSkeleton.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div
+        :class="$style['c-restaurantCard-skeleton']"
+        :style="`width: ${widthPercentage}%;`"
+        aria-hidden="true" />
```

Comment:
You might be able to use a css variable to prevent adding this style inline

Reply:
Yeah definitely could - would I still have to create an inline style to declare the width variable? Happy to do it but unsure what benefits that brings over a direct width declaration

Reply:
as discussed will leave as is for now and potentially switch to a CSS var in a future PR :) thanks!